tests/storage/sync_storage.rs

```diff
@@ -150,10 +151,25 @@ impl SyncStorage {
     }
 
     pub fn resolve_lock(&self, ctx: Context, start_ts: u64, commit_ts: Option<u64>) -> Result<()> {
+        let mut txn_status = HashMap::default();
+        txn_status.insert(start_ts, commit_ts.unwrap_or(0));
         wait_op!(|cb| {
-            self.store
-                .async_resolve_lock(ctx, start_ts, commit_ts, cb)
-                .unwrap()
+            self.store.async_resolve_lock(ctx, txn_status, cb).unwrap()
+        }).unwrap()
+    }
+
+    pub fn resolve_lock_batch(
+        &self,
+        ctx: Context,
+        start_ts: Vec<u64>,
```

Comment:
It's a little weird to split `start_ts` and `commit_ts`, I perfer to use `Vec<(u64,u64)>` instead.